install-server.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -e
+
+readonly ARGS=( "$@" )
+readonly PROGNAME="./install-server.sh"
+
+usage() {
+    cat <<- EOF
+    usage: $PROGNAME options
+
+    Install Trento Server
+
+    OPTIONS:
+       -p --private-key         the private key used by the runner to connect to the hosts
+       -h --help                show this help
+
+
+    Example:
+       $PROGNAME --private-key ./id_rsa_runner
+EOF
+}
+
+cmdline() {
+    local arg=
+    for arg
+    do
+        local delim=""
+        case "$arg" in
+            --private-key)  args="${args}-p ";;
```

Comment:
It didn't work for me. It has prompted "Please provide the path of the runner private key:" even when the key was provided.